Runtime/Scripts/RGBotConfigs/RGStateProviders/RGStatePlatformer2DPlayerStatsProvider.cs

```diff
@@ -0,0 +1,35 @@
+namespace RGBotConfigs.RGStateProviders
+{
+    public interface RGStatePlatformer2DPlayerStatsProvider
+    {
+        /**
+         * <summary> Get the current horizontal velocity of the player </summary>
+         */
+        public float Velocity();
+
+        /**
+         * <summary> Get the max horizontal velocity of the player </summary>
+         */
+        public float MaxVelocity();
+        
+        /**
+         * <summary> Get the current height that the player can jump </summary>
+         */
+        public float JumpHeight();
```

Comment:
Obviously this is all still work in progress, but the intent here was to try to give the bot information about pathing that may not be immediately obvious in the current tick.

In other words, right now the game may say that I can jump 5 tiles high.  But if I know the max jump height is 9 tiles high, there must be some item somewhere that increases my jump height and may allow me to get places I couldn't otherwise reach.

In the long run we may model this differently, but this is where I was in the iteration cycle when aaron suggested we wrap up the experiment for now.